downloads/index.md

```diff
@@ -244,7 +244,7 @@ meta_descr: downloads.intro
                             <th>{% t downloads.sourcecode %}</th>
                         </tr>
                         <tr>
-                            <td><img class="small" src="/img/cakewallet.png" width="172" height="202" loading="lazy" alt="Cake Wallet Logo"><a href="https://cakewallet.com/">Cake Wallet</a></td>
+                            <td><img class="small" src="/img/monerocom-wallet.png" width="172" height="202" loading="lazy" alt="Monero.com Wallet Logo"><a href="https://monero.com/wallets">Monero.com Wallet</a></td>
```

Comment:
```suggestion
                            <td><img class="small" src="/img/monerocom-wallet.png" width="172" height="202" loading="lazy" alt="Monero.com Wallet Logo"><a href="https://cakewallet.com">Monero.com Wallet</a></td>
```
Only issue is monero.com seems to be behind cloudflare, while cakewallet.com is not. Also cakewallet.com has fdroid link.
Cakewallet.com and monero.com/wallets both offer cake and monero.com